cpp/src/utilities/cython.cpp

```diff
@@ -27,28 +28,161 @@ namespace cython {
 // Populates a graph_container_t with a pointer to a new graph object and sets
 // the meta-data accordingly.  The graph container owns the pointer and it is
 // assumed it will delete it on destruction.
-//
-// FIXME: Should local_* values be void* as well?
 void populate_graph_container(graph_container_t& graph_container,
-                              legacyGraphTypeEnum legacyType,
-                              raft::handle_t const& handle,
-                              void* offsets,
-                              void* indices,
+                              raft::handle_t& handle,
+                              void* src_vertices,
+                              void* dst_vertices,
                               void* weights,
-                              numberTypeEnum offsetType,
-                              numberTypeEnum indexType,
+                              void* vertex_partition_offsets,
+                              numberTypeEnum vertexType,
+                              numberTypeEnum edgeType,
                               numberTypeEnum weightType,
-                              int num_vertices,
-                              int num_edges,
-                              int* local_vertices,
-                              int* local_edges,
-                              int* local_offsets,
+                              int num_partition_edges,
```

Comment:
I think this should be size_t as well.

Reply:
The problem with using `size_t` here is that since `num_partition_edges` is passed to the `edgelist_t` ctor directly and is expecting it as an `edge_t`, it results in a "narrowing conversion" which doesn't compile, but I agree that `size_t` or even `edge_t` would be better.

I will address this in PR #1178